Sprint-2/errors/0.js

```diff
@@ -2,8 +2,13 @@
 
 // call the function capitalise with a string input
 // interpret the error message and figure out why an error is occurring
+//
+//Answer: SyntaxError: Identifier 'str' has already been declared
+//change the variable name inside the function body
 
 function capitalise(str) {
-  let str = `${str[0].toUpperCase()}${str.slice(1)}`;
-  return str;
+  let newStr = `${str[0].toUpperCase()}${str.slice(1)}`;
```

Comment:
Do you even need the variable? Could you return the template literal expression directly?

Reply:
No, I do not need the variable. I changed my code based on review